README.md

```diff
@@ -1 +1,288 @@
-# Project-Week-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] I have seen a broken image for the logo in your readme. You are encouraged to replace this logo with your own! Otherwise, you can also remove it. 👍🏾 
![logo](https://github.com/Mahdi-Niazi/Project-Week-1/assets/99656687/022c558f-7b76-4a8d-9363-f54e4b75f235)